### PR TITLE
Minor aggregation syntax changes

### PR DESCRIPTION
**Pull request by ssanjay1**
Rename group to $group and remove values parameter from agg results

### CODE REVIEW

File: packages/client/src/object/aggregateOrThrow.test.ts [hunk 1]
```diff
@@ -165,16 +165,16 @@ describe("aggregateOrThrow", () => {
...
         },
         groupBy: {
           text: "exact",
         },
       },
     );
     expectType<Array<any>>(grouped);
-    expectType<string | undefined>(grouped[0].group.text);
-    expectType<number>(grouped[0].values.text.approximateDistinct);
+    expectType<string | undefined>(grouped[0].$group?.text);
```
Comment (ssanjay1) [inline packages/client/src/object/aggregateOrThrow.test.ts:176]: I had to optional chain here to get the tests to pass, but I have no idea why my changes did that

Comment (ssanjay1) [reply packages/client/src/object/aggregateOrThrow.test.ts:176]: For context ,the error tests gave was `cannot read properties of undefined` on group and id, which is why I optional chained it

Comment (ssanjay1) [reply packages/client/src/object/aggregateOrThrow.test.ts]: Actually, was able to fix this

File: packages/client/src/object/aggregateOrThrow.ts [hunk 2]
```diff
@@ -85,8 +85,8 @@ export async function aggregateOrThrow<
   const ret: AggregationResultsWithGroups<Q, AO["select"], any> = result.data
     .map((entry) => {
       return {
-        group: entry.group as any,
-        values: legacyToModernSingleAggregationResult(entry),
+        $group: entry.group as any,
+        ...legacyToModernSingleAggregationResult(entry),
       };
     }) as any; // fixme
```
Comment (ssanjay1) [inline packages/client/src/object/aggregateOrThrow.ts:91]: Is there a good way to fix this, besides doing something like `as AggregationResultsWithGroups<Q, AO["select"], any>`?

**Review by mfedderly — APPROVED** [review]

File: packages/client/src/object/aggregateOrThrow.ts (hunk 2 above)
Comment (ssanjay1) [reply packages/client/src/object/aggregateOrThrow.ts:91]: Ok I won't block merge on it, will think about it as I do other agg stuff potentially

**Pull request merged**